public/app/features/search/page/components/FolderView.tsx

```diff
@@ -58,41 +58,33 @@ export const FolderView = ({ selection, selectionToggle, onTagSelected, tags, hi
     return folders;
   }, []);
 
-  if (results.loading) {
-    return <Spinner />;
-  }
-  if (!results.value) {
-    return <div>?</div>;
-  }
+  const renderResults = () => {
+    if (results.loading) {
+      return <Spinner className={styles.spinner} />;
+    } else if (!results.value) {
+      return <Alert className={styles.error} title={results.error ? results.error.message : 'Something went wrong'} />;
```

Comment:
Nit: 🤔 I am unsure if showing this alert is a good way of handling errors for users, do we have this pattern elsewhere? maybe @amy-super has a better suggestion?

Reply:
@axelavargas yeah i just put something that would be better than a question mark 😅  i'll merge for now and we can discuss how to improve it since the tests will be pretty much the same 👍